source/sourcePane.js

```diff
@@ -151,14 +143,23 @@ const thisPane = {
         textArea.value = desc
 
         setUnedited()
-        var contentType, allowed
-        let rrr = kb.any(response.req, kb.sym('http://www.w3.org/2007/ont/link#response'))
-        if (rrr) {
-          contentType = kb.anyValue(rrr, UI.ns.httph('content-type'))
-          allowed = kb.anyValue(rrr, UI.ns.httph('allow'))
-          eTag = kb.anyValue(rrr, UI.ns.httph('etag'))
-          if (!eTag) console.log('sourcePane: No eTag on GET')
+        var contentType, allowed, eTag
+        if (response.headers && response.headers.get('content-type')) {
```

Comment:
contentType is already defined as a var in render, should it also be a var in refresh, or inherit -- unsure

Reply:
Mybad.  refresh() should inherit the outer level render() ones